test/debugger/test-debug-break-on-uncaught.js

```diff
@@ -91,13 +92,15 @@ function runScenario(scriptName, throwsOnLine, next) {
 
         client.reqContinue(function(error) {
           assert.ifError(error);
-          setTimeout(assertHasPaused.bind(null, client), 100);
+          interval = setInterval(assertHasPaused.bind(null, client), 10);
         });
       }
     );
   }
 
   function assertHasPaused(client) {
+    if (!exceptions.length) return;
+
     assert(exceptions.length, 'no exceptions thrown, race condition in test?');
```

Comment:
If I'm not mistaken, it's impossible for this assertion to fire given the check two lines above. So this line can probably be removed?